app/models/user.rb

```diff
@@ -88,4 +88,23 @@ def onboarded?
   def avatar_url
     "https://avatars2.githubusercontent.com/u/5337876?s=460&v=4"
   end
+
+  # TODO
+  def bio
+    "I am a user"
+  end
+
+  # TODO
+  def languages_spoken
+    %w[english spanish]
+  end
+
+  # TODO
+  def favorited_by?(_user)
+    false
+  end
+
+  def num_previous_mentor_sessions_with(_user)
```

Comment:
```suggestion
  # TODO
  def num_previous_mentor_sessions_with(_user)
```

Reply:
I think we should leave this todo here. This method isn't implemented yet.